recipes/maker/meta.yaml

```diff
@@ -32,6 +31,7 @@ requirements:
     - perl-inline-c
     - perl-perlio-gzip
     - perl-forks
+    - perl-list-moreutils ==0.15
```

Comment:
I added perl-list-moreutils version; should I also bump the build number? Linux was building successfully without it.

Reply:
This is strange. Maybe it is just a buildtime dependency and its installed inside our container? Can you try removing the run-time dependency. Also is there a need to pin the exact version?

Reply:
@bgruening perl-list-moreutils is more or less two different packages, from different authors at different times. I'm not sure 0.15 is exactly required, but it's the version listed in the `perl-forks` cpan dependencies and I decided it was known good. A later version fails (I'm re-running the test without pinning to get the error again). 

Not sure what's going on, so I'll also remove the run-time dep after that failure.

Reply:
I've reverted the breaking changes (removing the run-time dependency and version-locking to a known-good version because newer versions were conflicting). It's now breaking on the linux circleci builds, so it's not a mac specific issue. The conflicting errors from the [circleci build](https://circleci.com/gh/bioconda/bioconda-recipes/4574?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) were:

```
Undefined value assigned to typeglob at /opt/conda/conda-bld/maker_1520105724755/_t_env/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi/List/MoreUtils.pm line 94.
...
Undefined subroutine &List::MoreUtils::firstidx called at /opt/conda/conda-bld/maker_1520105724755/_t_env/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi/forks/signals.pm line 33.
Undefined subroutine &List::MoreUtils::minmax called at /opt/conda/conda-bld/maker_1520105724755/_t_env/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi/forks.pm line 1593.
Compilation failed in require at /opt/conda/conda-bld/maker_1520105724755/_t_env/bin/maker line 42.
BEGIN failed--compilation aborted at /opt/conda/conda-bld/maker_1520105724755/_t_env/bin/maker line 42.
```  

It's caused by `perl-forks` trying to use a method in perl-list-moreutils that I suspect is present in the older version but not in the newer one, so the only explanation I can think of is that the last time this recipe was created the older version was used and the newer one wasn't present yet. 

So, unfortunately it seems like version locking and including as a run-time dep are the correct ways to handle it. :-/